### ISSUE TITLE
[Request] Docker Image for instant deploy

### ISSUE THREAD

**Issue by Chaoschaot232** [OrderedSet86/webnei-solid#1]
Hey 👋 ,

realy nice Idea and definitly worth setting it up. May I ask you for delivering this within a prepared docker image?
My root server runs mainly everything within docker containers started using Gitlab so it would be fine to have
a useable docker image for this :)

Keep up the great work :)

**Comment by OrderedSet86** [comment]
Yes, when this is deployed for beta release, it will probably be in a Docker image. Currently the app is in pre-alpha since all the functionality isn't done yet. Hoping for beta release in a week or two.
